TKGame/PowerUps/Components/FireStone PowerUps/C_FireStone_PrimaryAttack.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Input;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TKGame.BackEnd;
+using TKGame.Components.Interface;
+using TKGame.Players;
+using TKGame.PowerUps.RelatedEntities;
+using TKGame.Status_Effects;
+
+namespace TKGame.PowerUps.Components.FireStonePowerUps
+{
+    public class C_FireStone_PrimaryAttack : IPrimaryAttackComponent
+    {
+        #region Constants
+        private readonly float SHOCKED_DURATION = 8f;
+        private readonly float SHOCKED_TICK_INTERVAL = 0.5f;
+        private readonly float SHOCKED_DAMAGE_PER_TICK = 0f;
+        #endregion Constants
+        ComponentType IComponent.Type => ComponentType.AttackPrimary;
+        public string NameID { get; private set; }
+        public Rectangle HitBox { get; set; }
+        public AttackType AttackType { get; }
+        public bool isAttacking { get; private set; }
+
+        public C_FireStone_PrimaryAttack()
```

Comment:
Is this supposed to still be named this?

Reply:
I see shock stuff in the fire class so not certain why

Reply:
The fire effects were added to the potion item, so I just decided to make the firestone have the shocked effect.